src/utils/__tests__/fixtures/directingDepartments.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
This is not needed, we already have a mock for that in `src/utils/__tests__/fixtures/index.js`

Reply:
I prefer to have my fixtures explicitly defined and not calculated which adds unnecessary complexity, execution time, and possible points of failure.

Reply:
Will keep this construction. The categories_private will be shorten.